src/pages/landing/stocks/_headerSection.js

```diff
@@ -0,0 +1,396 @@
+import React from 'react'
+import styled from 'styled-components'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Flex } from 'components/containers'
+import StocksHeaderOverlayBg from 'images/svg/stocksHeaderOverlayBg.svg'
+import StocksHeaderOverlayBelowSectionBg from 'images/svg/stocksHeaderOverlayBelowSectionBg.svg'
+import StocksHeaderBelowSection from 'images/svg/stocksHeaderBelowSection.svg'
+import stocksHeaderBelowSectionM from 'images/svg/stocksHeaderBelowSection-m.svg'
+import Show from 'components/containers/show'
+import device from 'themes/device.js'
+import { localize } from 'components/localization'
+import { Header, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+
+const query = graphql`
+    query {
+        mainHeaderImage: file(relativePath: { eq: "stocksMainHeaderImg.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const MainWrapper = styled(Flex)`
+    background-color: var(--color-white);
+    flex-direction: column;
+    overflow: hidden;
+`
+const HeaderWrapper = styled(Flex)`
+    position: relative;
+    overflow: hidden;
+    background-color: var(--color-black-8);
+    clip-path: polygon(0 0, 100% 0, 100% 100%, 0 calc(100% - 5vw));
+    @media ${device.tablet} {
+        clip-path: polygon(0 0, 100% 0, 100% 92%, 0% 100%, 0 calc(100% - 5vw));
+    }
+`
+
+const TopHeaderWrapper = styled(Flex)`
+    position: relative;
+    height: 680px;
+    width: 100%;
+    max-width: 1200px;
+    padding: 0 4rem;
+    z-index: 2;
+
+    @media ${device.desktopL} {
+        max-width: 1600px;
+    }
+    @media ${device.laptopM} {
+        height: 680px;
```

Comment:
I don't think this is required right? As you declared it above.

Reply:
Omitted in the latest commit..